test/contract/blackduck.e2e.test.ts

```diff
@@ -122,9 +108,9 @@ export function mockBlackduckParamsExcept(blackduckConstants: string[]) {
 }
 
 export function setAllMocks() {
+  let blackduck: string[] = []
   jest.spyOn(configVariables, 'getWorkSpaceDirectory').mockReturnValue(__dirname)
-  jest.spyOn(validator, 'validatePolarisInputs').mockReturnValueOnce(true)
-
+  jest.spyOn(validator, 'validateBlackDuckInputs').mockReturnValueOnce(blackduck)
```

Comment:
any reason we are changing from validateprolaris to validateblackduck?

Reply:
mocking respective validate methods.